src/app/payment/paymentMethod/PaymentMethodTitle.spec.tsx

```diff
@@ -272,4 +272,30 @@ describe('PaymentMethodTitle', () => {
         expect(component.find(CreditCardIconList).prop('selectedCardType'))
             .toEqual('visa');
     });
+
+    it('renders only Checkout.com APMs logos based on their gateway id', () => {
+        const checkoutcomAPMs = [
+            'sepa',
+            'oxxo',
+            'boleto',
+            'qpay',
+        ];
+
+        checkoutcomAPMs.forEach(apm => {
```

Comment:
rather than doing a foreach, you could create a testing component instead?
```js
const TestComponent = id => (<PaymentMethodTitleTest
                { ...defaultProps }
                method={ {
                    ...defaultProps.method,
                    method: PaymentMethodId.Checkoutcom,
                    id,
                } }
            />)

let component = <TestComponent id="sepa" />;
expect(component.find('[data-test="payment-method-logo"]'))
                .toEqual('sepaLogoUrl');
component = <TestComponent id="oxxo" />;
expect(component.find('[data-test="payment-method-logo"]'))
                .toEqual('oxxoLogoUrl');

// ...